src/main/kotlin/no/nb/bikube/core/model/Language.kt

```diff
@@ -0,0 +1,6 @@
+package no.nb.bikube.core.model
+
+data class Language(
+    val name: String?,
+    val catalogueId: String?
```

Comment:
Might be weird to call this catalogueId since they are NOT unique compared to the "regular" catalogueIds in newspapers? This is also the case in Publisher and PublisherPlace

Reply:
Changing to databaseI